rust/connlib/tunnel/src/device_channel/tun_darwin.rs

```diff
@@ -70,11 +70,7 @@ impl Tun {
         }
     }
 
-    pub fn new(
-        config: &InterfaceConfig,
-        dns_config: Vec<IpAddr>,
-        callbacks: &impl Callbacks,
-    ) -> Result<Self> {
+    pub fn new(config: &InterfaceConfig, callbacks: &impl Callbacks) -> Result<Self> {
```

Comment:
```suggestion
    pub fn new(_: &InterfaceConfig, _: &impl Callbacks) -> Result<Self> {
```

These are now unused, right?